src/serlo-editor-repo/plugin-text/hooks/use-formatting-options.tsx

```diff
@@ -67,13 +65,7 @@ const isRegisteredTextPlugin = (
   return option in textPluginsMapper
 }
 
-const toggleLinkAndFlag =
-  (setIsLinkNewlyCreated: SetIsLinkNewlyCreated) => (editor: SlateEditor) => {
-    toggleLink(editor)
-    setIsLinkNewlyCreated(true)
-  }
-
-const registeredHotkeys = (setIsLinkNewlyCreated: SetIsLinkNewlyCreated) => [
+const registeredHotkeys = () => [
```

Comment:
If you're sure we can remove the workaround, then this can be an array, it doesn't have to a function any more.